src/Model/Common/Position.java

```diff
@@ -0,0 +1,61 @@
+package Model.Common;
+
+/**
+ * Created by AldoBalderrama on 7/7/2016.
+ */
+public class Position
+{
+    //Attribute
+    private int positionId;
+    private String positionName;
+    private boolean enable;
+
+    //Property
+    public int getPositionId() {
+        return positionId;
+    }
+
+    public void setPositionId(int positionId) {
+        this.positionId = positionId;
+    }
+
+    public String getPositionName() {
+        return positionName;
+    }
+
+    public void setPositionName(String positionName) {
+        this.positionName = positionName;
+    }
+
+    public boolean isEnable() {
+        return enable;
+    }
+
+    public void setEnable(boolean enable) {
+        this.enable = enable;
+    }
+    //Constructor
+    public Position()
+    {
+        this.setPositionId(0);
```

Comment:
usually the constructor does not need to use the getters and setters they can access directly to the attributes